sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/WriteTables.java

```diff
@@ -215,6 +216,7 @@ public void processElement(
             firstPaneCreateDisposition,
             dynamicDestinations,
             destination);
+        LOG.debug("Use TableSchema for {}: {}", destination, tableSchema);
```

Comment:
I like "Fetching tableschema..." because it makes it clear we are getting it from BQ table, not from an input schema.